### PR TITLE
Updated HTMLEntriesEncode.pm: Fixes #3607

### LINKED ISSUE #3607: HTMLEntitiesEncode:
As of 2 minutes before posting this, the HTMLEntitiesEncode function is apparently replacing the entity output rather than displaying it raw. 

Example query: "Trademark entity" 

Expected Result:  &amp;#8482; (™)

Received result: ™ (™)

Tested on the latest versions of Chrome and Palemoon 

------
IA Page:  http://duck.co/ia/view/htmlentities_encode 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @nishanths

### PR DESCRIPTION
**Pull request by gautamkrishnar**
fixes #3607 
#### o/p:
screenshot 10

---

Instant Answer Page:  http://duck.co/ia/view/htmlentities_encode

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @nishanths

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/HTMLEntitiesEncode.pm [hunk 1]
```diff
@@ -6,6 +6,7 @@ use warnings;
 use DDG::Goodie;
 use Text::Trim;
 use HTML::Entities 'decode_entities';
+use HTML::Entities 'encode_entities';
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/HTMLEntitiesEncode.pm]: `use HTML::Entities qw(decode_entities encode_entities);`

**Comment by gautamkrishnar** [comment]
@GuiltyDolphin  Do I have to remove all the instances of html_enc()?
There's some more remaining,,,,

**Comment by GuiltyDolphin** [comment]
@gautamkrishnar Yeah, any remaining instances should be using `encode_entities`.

**Comment by gautamkrishnar** [comment]
@GuiltyDolphin  Done :smile:  :+1:

**Comment by GuiltyDolphin** [comment]
@gautamkrishnar Thanks! :dolphin:

**Pull request merged**